libs/npm/core/src/components/grid-table/Pager.tsx

```diff
@@ -92,43 +100,66 @@ export const Pager: React.FC<IPagerProps> = ({
 
   return (
     <styled.Pager>
-      <Button
-        name="firstPage"
-        variant={ButtonVariant.info}
-        onClick={() => gotoPage(0)}
-        disabled={!canPreviousPage}
-      >
-        <BiFirstPage />
-      </Button>
-      {showPageGroup && (
+      <Row justifyContent="center" className="button-container">
         <Button
-          name="previousPageGroup"
+          name="firstPage"
           variant={ButtonVariant.info}
-          onClick={() => previousPage()}
+          onClick={() => gotoPage(0)}
           disabled={!canPreviousPage}
         >
-          ...
+          <BiFirstPage />
         </Button>
-      )}
-      {rows()}
-      {showPageGroup && (
+        {showPageGroup && (
+          <Button
+            name="previousPageGroup"
+            variant={ButtonVariant.info}
+            onClick={() => previousPage()}
+            disabled={!canPreviousPage}
+          >
+            ...
+          </Button>
+        )}
+        {rows()}
+        {showPageGroup && (
+          <Button
+            name="nextPageGroup"
+            variant={ButtonVariant.info}
+            onClick={() => nextPage()}
+            disabled={!canNextPage}
+          >
+            ...
+          </Button>
+        )}
         <Button
-          name="nextPageGroup"
+          name="lastPage"
           variant={ButtonVariant.info}
-          onClick={() => nextPage()}
+          onClick={() => gotoPage(pageCount - 1)}
           disabled={!canNextPage}
         >
-          ...
+          <BiLastPage />
         </Button>
-      )}
-      <Button
-        name="lastPage"
-        variant={ButtonVariant.info}
-        onClick={() => gotoPage(pageCount - 1)}
-        disabled={!canNextPage}
-      >
-        <BiLastPage />
-      </Button>
+        <Show visible={!!manualPageSize}>
+          <Text
+            className="page-size"
+            width="5em"
```

Comment:
Always inline styling is horrible.  It's hard to override.  Put it in the styled component.